recipes/systemc/meta.yaml

```
@@ -0,0 +1,280 @@
+          - src/sysc/packages/boost/LICENSE_1_0.txt                                                                    # [not win]
+          - src/sysc/packages/qt/copyright.h                                                                           # [not win]
+        doc_url: https://ieeexplore.ieee.org/document/6134619                                                          # [not win]
+        dev_url: https://github.com/accellera-official/systemc                                                         # [not win]
+
+    # cmake build doesn't include the static lib if you ask for dynamic
+    #- name: libsystemc-static                                    # [not win]
+    #  files:                                                     # [not win]
+    #      - lib/libsystemc.a                                     # [not win]
+    - name: systemc-doc
+      files:
+          - Library/share/doc/systemc/  # [win]
+          - share/doc/systemc/          # [not win]
+      about:
+        home: https://systemc.org/
+        summary: 'The language for System-level design, modeling and verification'
+        description: |
+          Documentation installed by upstream build into $PREFIX along with the examples.
+          On Windows, the documentation is a bit light but the examples are in here.
+
+          SystemC(TM) addresses the need for a system design and verification language that spans hardware
+          and software. It is a language built in standard C++ by extending the language with the use of
+          class libraries. The language is particularly suited to model system's partitioning, to evaluate
+          and verify the assignment of blocks to either hardware or software implementations, and to
+          architect and measure the interactions between and among functional blocks. Leading companies in
+          the intellectual property (IP), electronic design automation (EDA), semiconductor, electronic
+          systems, and embedded software industries currently use SystemC for architectural exploration,
+          to deliver high-performance hardware blocks at various levels of abstraction and to develop
+          virtual platforms for hardware/software co-design. SystemC has been standardized by the Open
+          SystemC Initiative (OSCI) and Accellera Systems Initiative and ratified as IEEE Std. 1666(TM)-2011.
+        license: Apache-2.0
+        license_family: Apache
+        license_file:
+          - LICENSE
+          - NOTICE
+          - src/sysc/packages/boost/LICENSE_1_0.txt
+          - src/sysc/packages/qt/copyright.h
+        doc_url: https://ieeexplore.ieee.org/document/6134619
+        dev_url: https://github.com/accellera-official/systemc
+    - name: systemc-dev
+      build:                                                     # [not win]
+          run_exports:                                           # [not win]
+              - {{ pin_subpackage('libsystemc', exact=True) }}   # [not win]
```

A max_pin of 'x.x.x' or 'x.x' should be sufficient, no? There aren't build variants, and the ABI shouldn't be changing between build numbers of this package.

```suggestion
              - {{ pin_subpackage('libsystemc', max_pin='x.x.x') }}   # [not win]
```

---

Yes.  I'll fix this too.

---

Done in 
a896e1d